Background:

Bombay hospitality Ltd. operates a franchise model for producing exotic Norwegian dinners throughout New England. The operating cost for a franchise in a week (W) is given by the equation W = $1,000 + $5X, where X represents the number of units produced in a week. Recent feedback from restaurant owners suggests that this cost model may no longer be accurate, as their observed weekly operating costs are higher.

Objective:

To investigate the restaurant owners' claim about the increase in weekly operating costs using hypothesis testing.

Data Provided:

The theoretical weekly operating cost model: W = $1,000 + $5X

Sample of 25 restaurants with a mean weekly cost of Rs. 3,050

Number of units produced in a week (X) follows a normal distribution with a mean (μ) of 600 units and a standard deviation (σ) of 25 units


In [1]:
import scipy.stats as stats
import numpy as np

In [2]:
n=25 #sample size
x_bar=3050 #sample mean
W=1000+5*600 #theoretical weekly operating cost
sigma=5*25 # standard deviation of weekly cost
alpha=0.05 #significance level
CI=0.95 #confidence interval
# It is a one tailed t test

Null Hypothesis: The weekly operating costs are not higher than the model suggests.

Alternative Hypothesis: The weekly operating costs are higher than the model suggests.

In [3]:
t_statistics=(x_bar-W)/(sigma/np.sqrt(n)) # calculates t statistics
t_statistics

-38.0

In [4]:
critical_val=stats.norm.ppf(1-alpha) # calculates the critical value using z-table with 95% confidence interval
critical_val

1.6448536269514722

Since  the test statistics(-38.0) is smaller than the critical value(1.64) , we fail to reject Null hypothesis.

This means that the weekly operating costs are not higher than the model suggests.


Conclusion:

There is no strong evidence to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests.



Steps Invloved:
1. Stating Null and Alternative Hypothesis
2. Calculating test statistics
3. Determine the Critical value
4. Making a decision
5. Conclusion